scripts/url_text_extractor.py

```diff
@@ -0,0 +1,15 @@
+import requests
+from bs4 import BeautifulSoup
+
+def extract_text_from_url(url):
+    response = requests.get(url)
+    soup = BeautifulSoup(response.text, 'html.parser')
+    for br in soup.find_all("br"):
+        br.replace_with(" ")
```

Comment:
No. I want to extract all text in the website. But the text from each html tag must be separated by a string space.